src/components/LandingAnimation/LandingAnimation.js

```diff
@@ -105,27 +125,48 @@ const LandingAnimation = () => {
           debrisTwoRef.current,
           debrisThreeRef.current,
           debrisFourRef.current,
-        ]
+        ],
+        setAnimationPlay,
+        false
       );
-  });
+    }
+  }, [isAnimationPlaying]);
 
   return (
     <LandingAnimationWrapper>
       <StartButton
         ref={animationButtonRef}
         onClick={handelAnimationButtonClick}
       >
-        Press to say &apos;Hello!&apos;
+        <FormattedMessage id={animationButtonTextId} />
       </StartButton>
-      <StyledSpan ref={hiSpanRef}>Hi!</StyledSpan>
-      <StyledSpan ref={mySpanRef}>My</StyledSpan>
-      <StyledSpan ref={hiMySpanRef}>Hi! My</StyledSpan>
-      <StyledSpan ref={nameSpanRef}>name</StyledSpan>
-      <StyledSpan ref={hiMyNameSpanRef}>Hi! My name</StyledSpan>
-      <StyledSpan ref={isSpanRef}>is</StyledSpan>
-      <StyledSpan ref={hiMyNameIsSpanRef}>Hi! My name is</StyledSpan>
-      <StyledSpan ref={karolSpanRef}>Karol</StyledSpan>
-      <StyledSpan ref={hiMyNameIsKarolSpanRef}>Hi! My name is Karol</StyledSpan>
+      <StyledSpan ref={hiSpanRef}>
+        <FormattedMessage id="animation.hiSpan" />
```

Comment:
those `Span` is kinda shit - what if there will be needed to change the html content? 
Also - In my opinion they should be a `paragraphs`